e2e-tests/react-next/.env.example

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_WC_PROJECT_ID=
```

Comment:
best to add `NEXT_PUBLIC_PROVIDER_URL=https://testnet.fuel.network/v1/graphql` here